File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGDirection.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGDirection {
    YGDirectionInherit,
    YGDirectionLTR,
    YGDirectionRTL;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGDirection forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGDirection.kt
package app.cash.redwood.yoga.internal.enums

enum class YGDirection {
    YGDirectionInherit, YGDirectionLTR, YGDirectionRTL;

    fun getValue(): Int {
        return ordinal
    }

    companion object {
        fun forValue(value: Int): YGDirection {
            return values()[value]
        }
    }
}
